BootloaderCorePkg/Library/CpuExceptionLib/X64/CpuExceptionHandler.c

```diff
@@ -39,6 +41,18 @@ CommonExceptionHandler (
   }
 
   DEBUG ((DEBUG_ERROR, "Exception #%d from 0x%04X:0x%08X !!!\n", ExceptionType, (UINT32)Ptr[1], (UINT32)Ptr[0]));
+
```

Comment:
This part seems to be common for IA32 and X64.  Maybe we can move it out to up level so that we only need to maintain one copy.   But we can do it as separate patch I guess.  :)

Reply:
Makes sense to me to do this as a separate patch. :)